neqo-transport/src/connection/mod.rs

```diff
@@ -2664,7 +2666,7 @@ impl Connection {
             let limit = if path.borrow().pmtud().needs_probe() {
                 needs_padding = true;
                 debug_assert!(path.borrow().pmtud().probe_size() >= profile.limit());
-                path.borrow().pmtud().probe_size() - aead_expansion
+                path.borrow().pmtud().probe_size() - aead_expansion - Self::SCONE_INDICATION.len()
```

Comment:
Do I understand correctly, that we are effectively giving up 2 bytes of every outgoing MTU-sized datagram?

Reply:
Only for packets that we need to pad (Initial and any probing packet).  I figure that this is worth it.